schemas/internal_server.proto

```diff
@@ -85,9 +85,9 @@ message BookmarkPb { // Used for paginated responses
 
 message InternalScanPb {
     message TagScanPb {
-        repeated AttributePb attributes = 1;
-        optional google.protobuf.Timestamp earliest_create_time = 2;
-        optional google.protobuf.Timestamp latest_create_time = 3;
+        optional google.protobuf.Timestamp earliest_create_time = 1;
```

Comment:
This is fine for now, but after we release for Elevance we would need to ensure compatibility. 

P.S. Tech Huddle question: How can we ensure that we have automated contract tests.